.github/workflows/merge_to_main_production.yaml

```diff
@@ -24,6 +24,16 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Set Prod Version
+        run: |
+          TARGET_VERSION=`cat VERSION`
```

Comment:
Minor tweak - we'll need to update the workflow trigger so that it runs on `VERSION` file changes:
```yaml
  push:
    branches:
      - main
    paths:
      - ".github/workflows/merge_to_main_production.yaml"
      - "VERSION"
```

Reply:
Good catch, updated! Thanks 👍🏾